app/models/concerns/reports/post_edits.rb

```diff
@@ -89,7 +90,8 @@ def report_post_edits(report)
         end
       end
 
-      builder.where("editor.id > 0 AND editor.id != author.id")
+      builder.where("editor.id = ?", editor_id) if editor_id
```

Comment:
This is a trivial thing but I think it would be more readable as a multi line if/else:

```ruby
if editor_id
  builder.where("editor.id = ?", editor_id)
else
  builder.where("editor.id > 0 AND editor.id != author.id")
end
```

I like putting conditions on the end of lines but in this case you're doing it on two lines in succession with opposite conditions.